quickstart_docker/Dockerfile_quickstart

```diff
@@ -0,0 +1,33 @@
+ARG BASE_CONTAINER=yikunkero/spark:latest
+
+FROM $BASE_CONTAINER as spark
+
+LABEL maintainer="Andrew Bauman"
+
+FROM spark as delta
+
+USER root
+
+ARG DELTA_CORE_VERSION="2.1.0"
+
+RUN pip install --no-cache-dir delta-spark==${DELTA_CORE_VERSION} \
+jupyterlab
+
+FROM delta as startup
+
+ARG NBuser=NBuser
+ARG GROUP=NBuser
+ARG WORKDIR=/opt/spark/work-dir
+ENV DELTA_PACKAGE_VERSION=delta-core_2.12:2.1.0

```

Comment:
Would it be possible to build a variable for this so we can change the Delta Lake version (and Scala version) in one place?

Reply:
If I'm interpreting the request correctly it isn't possible with this approach to building a docker image.  That change would have to be made at the base image level, per the example https://hub.docker.com/r/yikunkero/spark/tags

If we wanted to specify scala version we would have to either move back to a strategy of installing spark from source and pass the args there OR integrate the delta installs with the spark build (so in the base image).  

The good news is that since this entire image is based on the example of an official image (and will ultimately point to the official image) the build will always stay updated with latest.  If someone wanted to use a prior version they would just need to change the dockerfile to point to a different base spark (ie if they wanted a legacy version).

Please let me know if my explanation doesn't make sense or otherwise needs clarification.

Reply:
Ahh got it, I'll create an issue with the original base image in that case, thanks!